external/fv3fit/fv3fit/pytorch/graph_config.py

```diff
@@ -0,0 +1,64 @@
+import dataclasses
+from typing import Callable
+import torch.nn.functional as F
+import torch
+import torch.nn as nn
+from dgl.nn.pytorch import SAGEConv
+
+
+@dataclasses.dataclass
+class GraphNetworkConfig:
+    """
+    Attributes:
+        in_feats: number of input features
+        out_feats: number of output features
+        n_hidden: number of hidden channels
+        num_step: number of U-net blocks
+        aggregat: type of aggregator
```

Comment:
If there are only a couple options for aggregator, it would help to include them in the docstring.